opendebates/tasks.py

```diff
@@ -19,19 +20,28 @@ def update_recent_events():
     """
     logger.debug("update_recent_events: started")
 
-    votes = Vote.objects.select_related(
-        "voter", "voter__user", "submission").exclude(
-        voter=F('submission__voter')).order_by("-id")[:10]
-    submissions = Submission.objects.select_related(
-        "voter", "voter__user").order_by("-id")[:10]
+    try:
+        # No middleware on tasks, so this won't get set otherwise.
+        # Tell the DB router this thread only needs to read the DB, not write.
+        # We run this task frequently, so it's worthwhile.
+        set_thread_readonly_db()
```

Comment:
If/when this needs to be done elsewhere, might be nice to refactor this to use context manager or decorator syntax.